src/renderer.js

```diff
@@ -159,3 +161,11 @@ function loadCurrentDayTasks(dayNumber, monthNumber) {
     }
   });
 }
+
+//TODO implement it similar to the loadTasks foreach loop but on clicking the calendar
+function loadTomorrowTasks() {}
+
+function appendTaskToTomorrowList(task) {
```

Comment:
Separated this functionality (with the help of ChatGPT) because for some reason when the same code but inside the loadTasks() function did not work and nothing was being rendered in the right column